index.js

```diff
@@ -1,184 +1,286 @@
+	}
 }
-
+/**
+ * "If the flag is blue and the blue flag is disabled, return. If the mode is 1 and the current mode is
+ * 1, send a request to the Pixoo64 to change the GIF. If the request fails, log an error. Otherwise,
+ * return. Otherwise, set the flag path to the path of the flag and set the src of the #digiflag
+ * element to the flag path."
+ * </code>
+ * @param flag - the flag to change to
+ * @param mode - 1 = Pixoo64, 2 = DigiFlag
+ * @returns a Promise.
+ */
 async function changeGif(flag, mode) {
-    if(flag === "blue" && disabledBlueFlag) return;
-    if(mode === 1 && currentMode === 1) {
-        res = await httpGet(`http://localhost:9093/pixoo/${flag}/${currentTheme}/${pixooIP}`)
-        if(res !== 'OK') {
-            console.log('Failed to change GIF on Pixoo64');
-        }
-        return;
-    }
-    flagPath = getGifPath(flag);
-    $('#digiflag').prop('src', flagPath)
+	if (flag === "blue" && disabledBlueFlag) return;
+	if (mode === 1 && currentMode === 1) {
+		const res = await httpGet(
+			`http://localhost:9093/pixoo/${flag}/${currentTheme}/${pixooIP}`
+		);
+		if (res !== "OK") {
+			console.log("Failed to change GIF on Pixoo64");
+		}
+		return;
+	}
+	const flagPath = getGifPath(flag);
+	$("#digiflag").prop("src", flagPath);
 }
 
 function linkSuccess() {
-    $('#tagLink').addClass('text-bg-success')
-    $('#tagLink').removeClass('text-bg-primary')
-    $('#tagLink').removeClass('text-bg-warning')
-    $('#tagLink').text('Connected to F1MV')
-    $('#edit_hostInfo').remove()
-    $('#LinkF1MV').remove()
-    $('#infotag').text('')
-    $('#infolink').text('')
-
-    $('#select_theme').append(`
+	$("#tagLink").addClass("text-bg-success");
+	$("#tagLink").removeClass("text-bg-primary");
+	$("#tagLink").removeClass("text-bg-warning");
+	$("#tagLink").text("Connected to F1MV");
+	$("#edit_hostInfo").remove();
+	$("#LinkF1MV").remove();
+	$("#infotag").text("");
+	$("#infolink").text("");
+	$("#select_theme").append(`
         <div id="themes">
 
         </div>
         <button type="button" id="nextTheme" class="btn btn-success" disabled>Next</button>
-    `)
-    let theme
-    for (let _i = 0; _i < themes.length; _i++) {
-        theme = themes[_i];
-        $('#themes').append(`
+    `);
+	let theme;
+	for (let _i = 0; _i < themes.length; _i++) {
+		theme = themes[_i];
+		$("#themes").append(`
             <div class="form-check" id="window">
                 <input class="form-check-input theme" type="radio" name="theme" id="${theme.id}">
                 <label class="form-check-label theme" for="${theme.id}">
                     ${theme.name}
                 </label>
             </div>
-        `)
-    }
-    $('.theme').click((e) => {
-        selectTheme(e.target.id * 1)
-    })
-    $('#nextTheme').click(() => {
-        $('#select_theme').remove()
-        $('#select_device').append(`
+        `);
+	}
+	$(".theme").on("click", (e) => {
+		selectTheme(e.target.id * 1);
```

Comment:
Why * 1 ?

Reply:
I never added that lol. That has been there since as far as I can tell v1.1.0-stable [ selectTheme(e.target.id * 1)](https://github.com/LapsTimeOFF/DigiFlag_F1MV/blob/a8aa4d5d8a9fca1919a6a01cc675557fcc0ab31c/index.js#L177)

Reply:
Okay, idk why I've put that